testing/dataflowexample/travis-test.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 WORKING_DIR=$(cd $(dirname "$0") && pwd)
 
-JSR308=$(cd $WORKING_DIR/../../../ && pwd)
+export JSR308=$(cd $WORKING_DIR/../../../ && pwd)
```

Comment:
this doesn't need to be exported, JSR308 is the directory that is the parent of CFI, and should always be set as a relative path

Reply:
Thanks for your help. I am confused about it. Because the variable JSR308 is only valid in shell script (without export), when we call dljc, we try to read the value of JSR308 from environment by using `os.environ["JSR308"]` , there will be an KeyError. If we cannot export it to make it exist in environment variables, what should we do to set JSR308 into environment variable?

Reply:
In the setup instructions for CF and CFI circa 2017 or earlier, you had to declare JSR308 as an bash env variable in your bash_rc. CF has since made all of such paths relative in its scripts. CFI might be (there can still be some hard coded paths in opprop/CFI). DLJC hasn't been updated to follow this paradigm, but the setup scripts should put it in $JSR308/dljc. Feel free to file a PR to https://github.com/opprop/do-like-javac to update it to use relative paths.

Reply:
Thanks Jeff, I will do it.